UNRELEASED.md

```diff
@@ -8,6 +8,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Breaking changes
```

Comment:
I don't think this is a breaking change, there's no action that consumers need to take, right? Unless I'm misunderstanding something I think this is more of an `Enhancement`

Reply:
Thanks, I updated this to be under `Enhancement`